doc/safety.md

```diff
@@ -0,0 +1,57 @@
+# SECURITY
+
+This document describes the security recommendations and issues that can be helpful for users of Bitcoin Core.
+
+
+Download Bitcoin Core only from below links, verify the integrity of the download before using:
+
+https://bitcoincore.org/en/download/
+
+https://github.com/bitcoin/bitcoin/releases
+
+Downloading Bitcoin Core from untrusted sources can result in using a [malware](https://bitcoin.stackexchange.com/a/107738/) that can steal your bitcoin and compromise your computer.
+
+### P2P
+
+Using non-default ports and non-listening nodes can improve security.
+
+DNS seeds are used for bootstrapping the P2P network. If one of the domains used for default DNS seeds is hacked, this can impact nodes. A node checks 3 seeds at a time, so it would require coordination for a malicious seed to affect the network. You can use own DNS seed for bootstrapping.
+
+Difference in DNS seed and Seed node: https://bitcoin.stackexchange.com/a/17410/
+
+Having multiple connections to the network is fundamental to safety against eclipse attacks. If a user configured the node to only have outbound block-relay-only connections, they wouldn't be participating in addr relay, so wouldn't have a diverse addrman to identify candidates to connect to. It is suggested to have at least 10 outbound connections.
+
+`assumevalid=1` skips validation for scripts. All scripts in transactions included in blocks that are ancestors of the _assumevalid_ block are assumed to be valid. While this does introduce a little bit of trust, people can independently check that block is part of the main chain.
```

Comment:
The only way to independently check the block is part of the main chain, is to run without assumevalid...

Reply:
This commit is outdated. Please suggest changed in latest commit if I could improve something in this line.